dojo/db_migrations/0220_system_settings_disclaimer_notif.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 5.0.8 on 2024-09-12 18:22
+
+from django.db import migrations
+
+
+def copy_notif_field(apps, schema_editor):
+    system_settings_model = apps.get_model('dojo', 'System_Settings').objects.get()
+    if system_settings_model.disclaimer_notifications:
+        system_settings_model.disclaimer_reports = system_settings_model.disclaimer_notifications
+        system_settings_model.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dojo', '0219_system_settings_disclaimer_notif'),
+    ]
+
+    operations = [
+        migrations.RunPython(copy_notif_field),
```

Comment:
It would be ideal to have a "reverse" operation for this as well if possible. Even if is just a [noop](https://docs.djangoproject.com/en/5.1/ref/migration-operations/#django.db.migrations.operations.RunPython.noop)

Reply:
Good idea. Thank you. It would also be good to handle these in other (future) cases.